indico/htdocs/js/indico/widgets/protection_widget.js

```diff
@@ -29,16 +34,16 @@
         }, options);
 
         var inputs = $('input[name=' + options.fieldName + '][id^=' + options.fieldId + ']');
+        var $enableAclLink = $('.enable-acl-link');
```

Comment:
I know, it was my first intention but `ACLLink` looked weird to me because of the `LL` so a went for `Acl`. Anyway, not a big deal, I can change it.